classes/class-twenty-twenty-one-customize.php

```diff
@@ -146,6 +146,27 @@ public function register( $wp_customize ) {
 					)
 				)
 			);
+
+			$wp_customize->add_setting(
+				'respect_user_color_preference',
+				array(
+					'capability'        => 'edit_theme_options',
+					'default'           => true,
+					'sanitize_callback' => function( $value ) {
+						return (bool) $value;
+					},
+				)
+			);
+
+			$wp_customize->add_control(
+				'respect_user_color_preference',
+				array(
+					'type'        => 'checkbox',
+					'section'     => 'colors',
+					'label'       => esc_html__( 'Support visitor\'s color scheme preference', 'twentytwentyone' ),
```

Comment:
I feel if the wording were reversed, it would be more clear:

**Label**
Support Dark Mode

**Description**
By checking this box, your site will use a dark color palette when the visitor's system appearance is set to prefer dark: _link to more info about accessibility / system appearance settings_

Reply:
I'd like to suggest a slightly different label for this setting... `Respect visitor's device light/dark mode settings`.
* It makes it easier to understand that _"hey, users have their preferences, perhaps you should respect them"_.
* The use of "device" in there immediately makes me think of my phone, and it will be easier for someone non-technical to make the connection with the light/dark setting they have on their own device.

Reply:
I think we at least need to keep it at this shorter length. "By checking this box" it sounds a little bit like a users agreement =D